XMLtoMD/GuidelineXmlToMD/GuidelineXmlFileReader.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Xml.Linq;
+
+namespace GuidelineXmlToMD
+{
+    static class GuidelineXmlFileReader
+    {
+        public const string _Guideline = "guideline";
```

Comment:
Would be nice if these constants were put in their own static class. Since these are related to the structure of the XML and should be put in a class with a name that reflects that.

An even cleaner option is to use `xsd` to generate the XML classes/serializer so that you don't need to do it yourself.
You can find the doc on that [here](https://docs.microsoft.com/en-us/dotnet/standard/serialization/xml-schema-definition-tool-xsd-exe)